pepper-apis/src/main/java/org/broadinstitute/ddp/util/FilterUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.ddp.util;
```

Comment:
I find it a bit sad that the `ddp.util` package seems to have this gravitational attraction that pulls everything in, so a whole bunch of stuff ends up there. Can we move this `FilterUtil` to the `ddp.filter` package instead? You can remove the `import ... ExcludePathFilterWrapper` statement below as a bonus.

Reply:
OK. Have done a bit of refactoring to make code go from  😢 to 😄